Fix:

aafire.c:55:7: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

aafire.c:117:12: error: implicit declaration of function 'rand' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

aainfo.c:12:2: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

aakbdreg.c:37:8: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]

aamoureg.c:33:8: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]

aaregist.c:38:8: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]

aasavefont.c:10:2: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

aatest.c:8:5: error: implicitly declaring library function 'strcpy' with type 'char *(char *, const char *)' [-Werror,-Wimplicit-function-declaration]

aatest.c:11:2: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

--- src/aafire.c.orig	2001-04-26 10:30:03.000000000 -0500
+++ src/aafire.c	2020-09-22 02:33:54.000000000 -0500
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h>
 #include "aalib.h"
 
 #define XSIZ aa_imgwidth(context)
--- src/aainfo.c.orig	2001-04-26 09:37:31.000000000 -0500
+++ src/aainfo.c	2020-09-22 02:26:31.000000000 -0500
@@ -1,3 +1,4 @@
+#include <stdlib.h>
 
 #include "aalib.h"
 #include "aaint.h"
--- src/aakbdreg.c.orig	2020-09-22 02:24:57.000000000 -0500
+++ src/aakbdreg.c	2020-09-22 02:31:43.000000000 -0500
@@ -1,4 +1,5 @@
 #include <stdlib.h>
+#include <string.h>
 #include "config.h"
 #include "aalib.h"
 #include "aaint.h"
--- src/aamoureg.c.orig	2020-09-22 02:24:57.000000000 -0500
+++ src/aamoureg.c	2020-09-22 02:29:48.000000000 -0500
@@ -1,4 +1,5 @@
 #include <stdlib.h>
+#include <string.h>
 #include "config.h"
 #include "aalib.h"
 #include "aaint.h"
--- src/aaregist.c.orig	2020-09-22 02:24:57.000000000 -0500
+++ src/aaregist.c	2020-09-22 02:28:46.000000000 -0500
@@ -1,4 +1,5 @@
 #include <stdlib.h>
+#include <string.h>
 #include "config.h"
 #include "aalib.h"
 #include "aaint.h"
--- src/aasavefont.c.orig	2001-04-26 09:37:31.000000000 -0500
+++ src/aasavefont.c	2020-09-22 02:35:34.000000000 -0500
@@ -1,3 +1,4 @@
+#include <stdlib.h>
 #include "aalib.h"
 int main(int argc, char **argv)
 {
--- src/aatest.c.orig	2001-04-26 09:37:31.000000000 -0500
+++ src/aatest.c	2020-09-22 02:32:19.000000000 -0500
@@ -1,3 +1,5 @@
+#include <stdlib.h>
+#include <string.h>
 #include "aalib.h"
 int main(int argc, char **argv)
 {
